app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -72,7 +69,6 @@ def update
     api :POST, '/organizations', N_('Create organization')
     param :name, String, :desc => N_("name"), :required => true
     param :label, String, :desc => N_("unique label")
-    param :description, String, :desc => N_("description")
```

Comment:
@orrabin I don't think I saw this when I tested out your changes. Where/when do you see this `Dont know how to merge Must be String with Must be String (NotImplementedError)` error?